### DISCUSSION TITLE
Interest in my potential new icon glyph (Tailscale logo)?

### DISCUSSION THREAD

**Discussion by willmortimer** [ryanoasis/nerd-fonts#1729] [Ideas]
![ts_glyph_screenshot](https://github.com/user-attachments/assets/2bf08060-df3f-438b-ac76-123fa1dbe033)

I made a Tailscale VPN logo for a project I was working on.  Here is it patched into JetBrainsMonoNFM. The dots are shaded with thin diagonal lines in the circle.

I didn't want to submit a pull request, as the contributing.md notes the scarcity of code points, but I can share the SVG if there would be any interest of incorporating it into the project. Would it be better to submit it to devicons repo?

**Comment by jth0** [comment]
I'd like this as I use it at home. I also want to figure out how to make one for twingate (tailscale competitor, used at work)

**Comment by superlemon57** [comment]
Up

**Comment by panda667** [comment]
I like it!

**Comment by luisnquin** [comment]
In my case, I generated the glyph by asking gpt-5.6-sol to create it for me. It was a fairly simple task since the design is just three rows of circles:

https://github.com/luisnquin/nixos-config/commit/20a1cbbd3f1f81551dbad7eb82f3871f686bc315

The result looks amazing.

<img width="1920" height="107" alt="image" src="https://github.com/user-attachments/assets/a4242d92-8700-4cc1-9fc9-18fb3f823aea" />
